src/components/modal/CommentWriteModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+
+import { usePostComment } from '@/hooks/queries/sns';
+
+import { css } from '../../../styled-system/css';
+import Button from '../common/Button';
+import Modal from '../common/Modal';
+import Textarea from '../common/Textarea';
+
+interface IProps {
+  onClose: () => void;
+  currentSnsId: number;
+}
+
+const CommentWriteModal = ({ onClose, currentSnsId }: IProps) => {
+  const [content, setContent] = useState('');
+
+  const { mutate: postComment } = usePostComment({ onClose });
+
+  const handleSubmit = () => {
+    postComment({ postId: currentSnsId, content });
+  };
+
+  return (
+    <Modal onClose={onClose} className={css({ width: 600 })}>
+      <div className={styles.container}>
+        <h2 className={styles.title}>댓글 작성</h2>
+        <Textarea value={content} setValue={setContent} rows={9} />
+        <div className={styles.button_container}>
+          <Button text="취소" variant="outlined" onClick={onClose} />
+          <Button text="완료" disabled={!content} onClick={handleSubmit} />
+        </div>
+      </div>
+    </Modal>
+  );
+};
+
+export default CommentWriteModal;
+
+const styles = {
+  container: css({
+    padding: 20,
+    display: 'flex',
+    flexDirection: 'column',
+    gap: 20,
+  }),
```

Comment:
```suggestion
  container: vstack({
    padding: 20,
    gap: 20,
  }),
```
패턴으로 더 단순화 할 수 있습니다!

Reply:
이거 바꿔야 될 부분이 많아서 다음에 수정해서 올리겠습니다